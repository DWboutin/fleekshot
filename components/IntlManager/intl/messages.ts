import en from "./en";
import fr from "./fr";

export enum IntlLocale {
  en = "en",
  fr = "fr",
}

export interface IntlMessages extends Record<string, string> {
  hello: string;
}

const messages: Record<IntlLocale, IntlMessages> = {
  en,
  fr,
};

export default messages;
