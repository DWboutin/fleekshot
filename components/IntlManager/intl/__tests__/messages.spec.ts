import messages from "../messages";

describe("intl messages", () => {
  it("should match snapshot", () => {
    expect(messages).toMatchSnapshot();
  });
});
