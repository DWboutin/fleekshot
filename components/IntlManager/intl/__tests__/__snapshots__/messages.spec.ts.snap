// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`intl messages should match snapshot 1`] = `
Object {
  "en": Object {
    "home": "Home",
    "httpErrors_1000": "Duplicate \\"{field}\\" field with \\"{value}\\" in database",
    "httpErrors_1001": "Field \\"{field}\\" {message}",
    "httpErrors_1002": "Sign in error - {message}",
    "httpErrors_field_confirmPassword": "confirm password",
    "httpErrors_field_name": "name",
    "httpErrors_field_password": "password",
    "httpErrors_field_username": "username",
    "profile_title": "Profile",
    "profile_uploadPicture": "Upload picture",
    "signInForm_button_submit": "Sign in",
    "signInForm_errors_isRequired": "Is required",
    "signInForm_errors_mismatch": "Password mismatch",
    "signInForm_errors_noUser": "No user was found",
    "signInForm_errors_notStrongEnough": "Is not strong enough",
    "signInForm_errors_tooLong": "Is too long",
    "signInForm_errors_tooShort": "Is too short",
    "signInForm_input_password_label": "Password",
    "signInForm_input_username_label": "Username",
    "signInForm_message_success": "You're now signed in and will be redirected",
    "signInForm_title": "Sign in",
    "signUpForm_button_submit": "Sign up",
    "signUpForm_errors_isRequired": "Is required",
    "signUpForm_errors_mismatch": "Password mismatch",
    "signUpForm_errors_notStrongEnough": "Is not strong enough",
    "signUpForm_errors_tooLong": "Is too long",
    "signUpForm_errors_tooShort": "Is too short",
    "signUpForm_input_confirmPassword_label": "Confirm password",
    "signUpForm_input_name_label": "Complete name",
    "signUpForm_input_password_label": "Password",
    "signUpForm_input_username_label": "Username",
    "signUpForm_message_success": "You're now signed up",
    "signUpForm_title": "Sign up",
  },
  "fr": Object {
    "home": "Accueil",
    "httpErrors_1000": "Le champ \\"{field}\\" avec la valeur \\"{value}\\" se trouve déjà dans la base de donnée",
    "httpErrors_1001": "Le champ \\"{field}\\" {message}",
    "httpErrors_1002": "Erreur de connexion - {message}",
    "httpErrors_field_confirmPassword": "confirm password",
    "httpErrors_field_name": "nom complet",
    "httpErrors_field_password": "password",
    "httpErrors_field_username": "nom d'utilisateur",
    "profile_title": "Profil",
    "profile_uploadPicture": "Téléverser",
    "signInForm_button_submit": "Se connecter",
    "signInForm_errors_isRequired": "Est requis",
    "signInForm_errors_mismatch": "Ne correspond pas",
    "signInForm_errors_noUser": "Aucun utilisateur trouvé",
    "signInForm_errors_notStrongEnough": "Est trop faible",
    "signInForm_errors_tooLong": "Est trop long",
    "signInForm_errors_tooShort": "Est trop court",
    "signInForm_input_password_label": "Mot de passe",
    "signInForm_input_username_label": "Nom d'utilisateur",
    "signInForm_message_success": "Vous êtes maintenant connectés et serez redirigé",
    "signInForm_title": "Connexion",
    "signUpForm_button_submit": "S'inscrire",
    "signUpForm_errors_isRequired": "Est requis",
    "signUpForm_errors_mismatch": "Ne correspond pas",
    "signUpForm_errors_notStrongEnough": "Est trop faible",
    "signUpForm_errors_tooLong": "Est trop long",
    "signUpForm_errors_tooShort": "Est trop court",
    "signUpForm_input_confirmPassword_label": "Confirmation du mot de passe",
    "signUpForm_input_name_label": "Nom complet",
    "signUpForm_input_password_label": "Mot de passe",
    "signUpForm_input_username_label": "Nom d'utilisateur",
    "signUpForm_message_success": "Vous êtes maintenant inscrits",
    "signUpForm_title": "Inscription",
  },
}
`;
